manifests/index.ts

```diff
@@ -34,7 +34,12 @@ const manifest: Manifest = {
   rpcUrl: env.NEXT_PUBLIC_RPC_URL,
   // Set to true if contracts transactions are sponsored for Cometh Connect users.
   // Contracts to sponsor are your ERC721, 0x exchange and either the wrapped native token contract or your ERC20
-  areContractsSponsored: true
+  areContractsSponsored: true,
+
+  fiatCurrency: {
+    symbol: "usd", // all currencies can be found in the currencies.ts file
+    format: "$", // symbol to display after the amount
```

Comment:
I think those should be currencyId and currencySymbol instead of symbol and format